learners/setup.md

```diff
@@ -2,31 +2,19 @@
 title: Setup
 ---
 
-## Getting ready
+### Getting Ready
```

Comment:
Just popping in to make a small note about this:

I would suggest to make all the main headings level 2 so that the main sections show up in the sidebar.